Data/Sys/GameSettings/GGV.ini

```diff
@@ -17,3 +17,10 @@ CPUThread = False
 VertexRounding = True
 # Needed for some FMVs.
 ImmediateXFBEnable = False
+
+[Gecko]
+$60fps (driving, BBBH, GESK)
+04109E74 38000001
```

Comment:
I must have missed in my previous reviews. Has this been confirmed to be region-free?